rhel7/overlays/stig_overlay.xml

```diff
@@ -1,4446 +1,2964 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <overlays xmlns="http://checklists.nist.gov/xccdf/1.1">
   <overlay disa="2235" owner="disastig" ownerid="RHEL-07-010010" ruleid="rpm_verify_permissions" severity="high">
-    <VMSinfo SVKey="86473" VKey="71849" VRelease="4">
-      <title>The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values.</title>
-    </VMSinfo>
     <VMSinfo SVKey="86473" VKey="71849" VRelease="4">
       <title>The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values.</title>
     </VMSinfo>
     <title>The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values.</title>
   </overlay>
   <overlay disa="2235" owner="disastig" ownerid="RHEL-07-010010" ruleid="rpm_verify_permissions" severity="high">
-    <VMSinfo SVKey="86473" VKey="71849" VRelease="4">
-      <title>The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values.</title>
-    </VMSinfo>
     <VMSinfo SVKey="86473" VKey="71849" VRelease="4">
       <title>The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values.</title>
     </VMSinfo>
     <title>The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values.</title>
   </overlay>
```

Comment:
There are still some duplicate key entries, such as this one for ``86473``.

Reply:
Thanks. Fixed at https://github.com/ComplianceAsCode/content/pull/5075.